### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by Vatyx**
Made documentation in the README obvious.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +25,13 @@ Given this, we don’t recommend using the the GDK for Unreal for games in the p
 **Who is it for?**<br>
 We recommend you start using the GDK if:
 
-* You don’t mind the rough edges and are interested in contributing to the GDK (see our [contributions]({{urlRoot}}/contributing) policy).
+* You don’t mind the rough edges and are interested in contributing to the GDK (see our [contributions](https://docs.improbable.io/unreal/latest/contributing) policy).
```
Comment (ElleEss) [inline README.md]: It's always struck me as odd that we say - you can contribute but...oh, you can't. :-)

Comment (ElleEss) [reply README.md]: Do you think it's worth adding "on the documentation website"?

That is: "...(see our [contributions](https://docs.improbable.io/unreal/latest/contributing) policy on the documentation website).

File: README.md [hunk 2]
```diff
@@ -53,21 +55,17 @@ The GDK and its starter projects are available on GitHub.
...
 In order to get access to this fork, you need to link your GitHub account to a verified Epic Games account, and to have agreed to Epic's license. You will not be able to use the GDK for Unreal without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
 
-## SpatialOS Documentation
-* For guidance on SpatialOS concepts see the documentation on the [SpatialOS website](https://docs.improbable.io/reference/latest/shared/concepts/spatialos). <br/>
-* For information on the SpatialOS GDK for Unreal, we recommend you use the [GDK documentation on the SpatialOS website](https://docs.improbable.io/unreal/pre-alpha/index) because the documentation is designed for the website, however, you can look at the [documentation on GitHub](https://github.com/spatialos/UnrealGDK/tree/master/docs).
-
 ## Installation and setup
-For prerequisites, installation and setup, see the [Installation and setup]({{urlRoot}}/setup-and-installing.md) documentation.
+For prerequisites, installation and setup, see the [Installation and setup](https://docs.improbable.io/unreal/latest/setup-and-installing) documentation.
```
Comment (ElleEss) [inline README.md]: DO you think it's worth adding:

"...see the [Installation and setup](https://docs.improbable.io/unreal/latest/setup-and-installing) on the documentation website.

Comment (ElleEss) [reply README.md]: Actually - would be nice for all the docs website links on here but it doesn't matter too much. :-)

File: README.md (hunk 1 above)
Comment (m-samiec) [reply README.md]: Discussed offline, we'll leave these for now.

**Pull request merged**